targets/rp2040.json

```diff
@@ -7,6 +7,7 @@
     "binary-format": "uf2",
     "uf2-family-id": "0xe48bff56",
     "rp2040-boot-patch": true,
+    "openocd-transport":"rp2040",
```

Comment:
It is this actually to do with this specific PR? Seems like something else.

Reply:
Yeah sorry, this was me testing picoprobe's debugging ability,